src/fs.rs

```diff
@@ -9,10 +9,34 @@ use tempfile;
 use errors;
 use errors::ResultChainExt;
 
-/// A safe scratchpad for tests to manipulate.
+/// A filesystem scratchpad for tests to manipulate.
```

Comment:
Not sure everyone understands what's the implications of "scratchpad", so maybe make it explicit? random name, gets cleaned up when it leaves scope.